rules/terraformrules/terraform_empty_list_equality.go

```diff
@@ -10,6 +10,7 @@ import (
 
 // TerraformEmptyListEqualityRule checks whether is there a comparison with an empty list
 type TerraformEmptyListEqualityRule struct{}
+type void struct{}
```

Comment:
Better to just use anonymous empty structs rather than inventing a `void` type like this